actor/builtin/paymentbroker/paymentbroker.go

```diff
@@ -64,6 +64,7 @@ type PaymentChannel struct {
 	Target         address.Address    `json:"target"`
 	Amount         *types.AttoFIL     `json:"amount"`
 	AmountRedeemed *types.AttoFIL     `json:"amount_redeemed"`
+	AgreedEol      *types.BlockHeight `json:"agreed_eol"`
```

Comment:
Can we state in a comment what the difference between this and `Eol` is, based on this code (which I am green to) they appear to be very similar. Is `AgreedEol` the original time the channel was set up for?

Reply:
Yep, `AgreedEol` is what the payer and payee agree upon either initially or in an extension. `Eol` is the actual end of life, which can be different in the case of a dispute.

I'll add a comment.